server/builtin/debug_clean.go

```diff
@@ -28,8 +28,7 @@ func (a *builtinApp) debugClean() handler {
 				Call: &apps.Call{
 					Path: pDebugClean,
 					Expand: &apps.Expand{
-						AdminAccessToken: apps.ExpandAll, 
-						Locale:           apps.ExpandAll,
+						Locale: apps.ExpandAll,
```

Comment:
I am confused as to how is the enforcement of admin-only is done now for debug clean

Reply:
I think just need to add `ActingUser: expand.Summary,`